app/controllers/workouts_controller.rb

```diff
@@ -31,17 +31,31 @@ def show
     }]
   end
 
+  def index
+    if params[:query].present?
+      sql_query = <<~SQL
+        movies.title ILIKE :query
```

Comment:
movies.title ... hehe
Does this need to be updated/deleted/commented out?

Reply:
Haha I don't even know what branch this is. Has been amended on my current branch. Any way of deleting this and I'll push again?

Reply:
Oh sorry no this is the right branch, the search method got changed so many times it broke I had to rewrite it so copied the one from the lecture in for reference. Yeah just comment that one out, the correct method should be underneath